### ISSUE TITLE
__init__.py breaks library autodiscovery in RED 0.8.3

### ISSUE THREAD

**Issue by Mario-P** [nokia/RED#189]
After upgrade Eclipse to Oxygen and RED to 0.8.3 libraries that have different class name and module name cannot be discovered by RED. See the example project and errors screenshots attached. It was working on Eclipse Neon with RED 0.8.0.

Project - [RED_Tests.zip](https://github.com/nokia/RED/files/1774644/RED_Tests.zip)
QuickFix error:
![quickfixerror](https://user-images.githubusercontent.com/22638505/36894485-c78b6b92-1e0b-11e8-9134-117ae03f7bd8.JPG)
Auto discover error:
![autodiscovererror](https://user-images.githubusercontent.com/22638505/36894503-da165a92-1e0b-11e8-94b0-7467bd68fdf5.JPG)

**Comment by kormbrek-rally** [comment]
I am having the same problem upgrading from 0.8.0 to 0.8.3.
I will add some observations, noting that my projects files are not in the root of the project folder (avoid red.xml being picked up by SCM, etc.):
```
<project root>
  \stuff
      \libs
      \tests
      __init__.py
```
When using relative path imports inside tests (Library    ../libs/mylib.py), if there is an `__init__.py` present in the stuff folder, it breaks with an error ImportError: No module named stuff.libs.mylib. If you delete the init file and do quick fix, it will find the library.

The parent folder always seems to be used. For example if you have (Library    libs.mylib), it fails with error ImportError: No module named stuff.libs.mylib. Why did RED add `stuff` to the package name?

It does not help to add the stuff folder to the Python path in red.xml, but adding it when running with robot works fine.

**Comment by kormbrek-rally** [comment]
Also, true to the title of this issue, RED cannot import pabotlib correctly anymore
Library    pabot.PabotLib
```
Additional info: RED python session problem. Following exception has been thrown by python service:
            Traceback (most recent call last):
              File "/var/folders/_f/yg8tx5197jggr236ps7ws8wr0000gp/T/RobotTempDir5344382908641882781/robot_session_server.py", line 38, in inner
                result['result'] = func(*args, **kwargs)
              File "/var/folders/_f/yg8tx5197jggr236ps7ws8wr0000gp/T/RobotTempDir5344382908641882781/robot_session_server.py", line 59, in inner
                return func(*args, **kwargs)
              File "/var/folders/_f/yg8tx5197jggr236ps7ws8wr0000gp/T/RobotTempDir5344382908641882781/robot_session_server.py", line 84, in inner
                return to_call(*args, **kwargs)
              File "/var/folders/_f/yg8tx5197jggr236ps7ws8wr0000gp/T/RobotTempDir5344382908641882781/robot_session_server.py", line 107, in inner
                return to_call(*args, **kwargs)
              File "/var/folders/_f/yg8tx5197jggr236ps7ws8wr0000gp/T/RobotTempDir5344382908641882781/robot_session_server.py", line 157, in get_classes_from_module
                return red_module_classes.get_classes_from_module(module_location)
              File "/private/var/folders/_f/yg8tx5197jggr236ps7ws8wr0000gp/T/RobotTempDir5344382908641882781/red_module_classes.py", line 32, in get_classes_from_module
                module = importlib.import_module(module_name)
              File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
                __import__(name)
              File "/Users/kevinormbrek/PythonVMs/RISC-Automated-Tests/lib/python2.7/site-packages/pabot/pabot.py", line 83, in <module>
                from . import pabotlib
            ValueError: Attempted relative import in non-package
```

**Comment by KrzysztofJozefowicz** [comment]
that's correct, even empty init will break autodiscovery. Internal builds already have it fixed.

**Comment by KrzysztofJozefowicz** [comment]
released in RED 0.8.4
